### ISSUE TITLE
Amélioration des JVN

### ISSUE THREAD

**Issue by yanek** [11rdp/modpack#19]
A la demande de @Nikkaaa, je suis en train de modifier les JVN. Au départ, il s'agissait juste de les passer fullscreen, mais autant en profiter pour les rendre plus réalistes.

Pour moi, c'est terminé, mais si je soumet quand même le ticket **si jamais vous avez des idées** ou des requêtes. Je vais essayer de créer les lampes torche IR qui vont avec, mais je ne suis pas sûr que le moteur d'Arma me laisse faire, donc je met ça de côté pour l'instant. 

![arma3_2017-03-19_21-17-01](https://cloud.githubusercontent.com/assets/5952366/24084575/f0e13066-0cec-11e7-97b0-2d8108fb21c0.jpg)

Dans la jungle avec une lampe torche :
![arma3_2017-03-19_21-35-19](https://cloud.githubusercontent.com/assets/5952366/24084578/f6ecbc0a-0cec-11e7-9f7d-48ba88350782.jpg)

En comparaison, les JVN vanilla : 
![arma3_2017-03-19_21-20-41](https://cloud.githubusercontent.com/assets/5952366/24084589/13e4d34c-0ced-11e7-9b75-4b98254b19b1.jpg)

**Comment by Sociohunter** [comment]
Ça a carrément plus de gueule. A voir le  rendu en jeu.
Par contre ce serait pas trop net ? Il me semble que c'est plus flou normalement. (Si quelqu'un a déjà regardé via des nvg, je suis curieux d'ailleurs)
Tu entends quoi par lampe torche IR ?

**Comment by 11RDP-LoupVert** [comment]
J'aime bien le rendu couleur que tu as obtenu, et avec les lunettes de dernière génération à quatre voies (panoramiques), c'est pas si improbable d'avoir un champ aussi large et cette netteté, mais pas forcément cette profondeur (de champ).

![](http://www.jumelle-vision-nocturne.fr/wp-content/uploads/2014/01/G%C3%A9n%C3%A9rations-vision-nocturne.jpg)


Avec des optiques de plus ancienne génération, c'est nettement plus "bruyant" et on distingue nettement la structure hexagonale du tube intensificateur sur les premières générations :
![](http://www.intas.org/Nachtsicht/struktur.jpg)
![](http://www.ecpad.fr/wp-content/gallery/commando_montagne/11_N2009-324P02-0004.jpg)
![](http://i43.tinypic.com/i2k60i.jpg)
![](http://i2.guns.ru/forums/icons/forum_pictures/001856/1856900.jpg)
![](http://4.bp.blogspot.com/-hGo82NyLBC4/UTz1vLYTpmI/AAAAAAAATuA/VcNqzsIp1as/s1600/nightvision.png)
![](https://sc01.alicdn.com/kf/HTB168usGXXXXXacXVXXq6xXFXXXk/30633/HTB168usGXXXXXacXVXXq6xXFXXXk.jpg)


Et elles sont beaucoup plus susceptible à l'éblouissement (un autre avantage du réducteur de son pour le tir de nuit )
![](http://www.defense.gouv.fr/var/dicod/storage/images/base-de-medias/images/air/actualites/images-2014/images-fevrier-2014/vision-sur-le-terrain-de-nuit-a-travers-des-jvn/2704305-1-fre-FR/vision-sur-le-terrain-de-nuit-a-travers-des-jvn.jpg)
![](https://i.ytimg.com/vi/Ec0J6zau7jY/maxresdefault.jpg)

Tu penses que ce serait possible de scripter le "rétro-éclairage" qui illumine le visage du porteur d'un halo vert ?
![](http://tpe-helicoptere.e-monsite.com/medias/images/jvn-bk-bis.jpg)

https://www.youtube.com/watch?v=sOrUP0Rq6Jg


La lampe torche IR, c'est peut-être le projecteur IR qui sert à courte portée pour la lecture de carte par exemple? (et te rend très visible, comme si tu tirais sur ta clope)

**Comment by yanek** [comment]
Effectivement, le bruit, le flou et la qualité de l'image de manière générale varient en jeu selon la génération de JVN utilisée (j'ai gardé celles définies par ACE comme base). Les captures d'écran ont été prises avec des Gen3 il me semble, peut-être 4. 

Concernant la distance de vue, techniquement il est possible de simuler ça avec un brouillard local qui ne s'affiche que lorsque les JVN sont activées. Mais on en a conclut avec @Nikkaaa que ce serait donner un avantage trop important aux IA qui ne verraient pas ce brouillard. N'ayant pas trouvé de solution suffisamment peu coûteuse en ressource, j'ai mis de côté pour le moment.

En ce qui concerne la torche IR, je pensais plus à quelque chose qu'on pourrait attacher sur l'arme. Quelque chose dans ce goût là : http://www.mountsplus.com/AR-15_Accessories/AR-15_Scope_Rings/SUR-M1.html
Normalement on a déjà tout ce qu'il faut pour lire la carte de nuit avec ACE. Il s'agit vraiment d'accentuer l'éclairage de l'environnement, par temps couvert, dans la jungle ou en intérieur, quand les conditions lumineuses ne permettent pas aux JVN seules d'avoir un rendu correct.

Pour l'illumination du visage, je vais regarder. Ça doit être possible, mais je sais pas quel est le coût en matière de performances.

**Comment by 11RDP-LoupVert** [comment]
Bien pensé, merci pour les détails Yanek.

Pour l'éclairage, vous aviez vu la Shoulder-mounted Light de Feint ? http://steamcommunity.com/sharedfiles/filedetails/?id=832551294
